llvm/lib/Transforms/Utils/ProfileVerify.cpp

```diff
@@ -20,8 +20,12 @@
 #include "llvm/IR/MDBuilder.h"
 #include "llvm/IR/ProfDataUtils.h"
 #include "llvm/Support/BranchProbability.h"
+#include "llvm/Support/CommandLine.h"
 
 using namespace llvm;
+static cl::opt<int64_t>
+    DefaultFunctionEntryCount("profcheck-default-fct-entry-count",
```

Comment:
nit: Spell out function? It seems strange to only choose to drop characters from one word in the option name.

Reply:
lol yes